### DISCUSSION TITLE
Burn In Prompt (Optional)

### DISCUSSION THREAD

**Discussion by Nitrowing** [rsxdalv/TTS-WebUI#235] [Q&A]
I can't find a decent explanation for what I should enter in this field - has anyone figured this out?

**Comment by rsxdalv** [comment]
You can leave it empty as well. It's good if you often generate something
that you use as a base for the next generation. For example, if you wanted
a train station voice, and you entered a station announcement so that the
model would "know" the context. But this might not work purely because the
model often does not understand or care.

**Reply by Nitrowing** [reply]
Ah, ok. I'm trying to get the voice to do a serious explanation and have entered "sombre"

**Comment by rsxdalv** [comment]
I see. Bark is trained directly on transcripts, so unfortunately it's very
rare that it will understand descriptions.

**Comment by Nitrowing** [comment]
So I would be better copy/pasting the original transcription of the voice I used for training?

**Comment by rsxdalv** [comment]
Could be. Note that bark can't generate the same phrase twice in a
sequence, if you use one phrase as the base of the same phrase. So if you
see the model becoming weird, that could be the reason.

**Comment by Nitrowing** [comment]
I can't even get it to repeat a generation. It seems to infect itself with the previous one. 
I'm finding this frustrating. I've been using Stable Diffusion for months with no big issues and was hoping your project, using a similar interface, wouldn't be too difficult to get my head around.
I give it an audio file, I gave it the transcription. I gave it the words to narrate.

**Comment by rsxdalv** [comment]
Stable diffusion has it's legwork done already, audio models are different.
I recommend starting with tortoise. I'll see what I can do to make bark
less annoying, but it's quite difficult to use at times.

**Comment by Nitrowing** [comment]
I tried and failed to install tortoise which is why I was trying rsxdalv

**Comment by rsxdalv** [comment]
Maybe we can fix the installation tomorrow.

**Comment by rsxdalv** [comment]
Any updates? Did you figure out tortoise or get another project working?

**Comment by Nitrowing** [comment]
Neither, I just tried a load more times and then gave up :(

**Reply by rsxdalv** [reply]
Oh, I hope that it will become more useful to people. Even now there's a lot of tools and knowledge missing to make it truly useful.
